docs/4-Prototyping.md

```diff
@@ -1,29 +1,170 @@
 # Prototyping
-## Overview
+
+A website prototype can be any mock-up or demo of what a website could look like when it goes live. A living prototype is a prototype that's actively used throughout the lifecycle of the website. Even after the site is deployed, the prototype lives on as an active demonstration of what the website looks like.
+
+The following features and conventions come together to allow the creation of a living prototype.
+
+## Conventions
+
+### Meta-Data
+
+Meta-Data is general data that helps describe the content you're presenting. Depending on the defining method, meta-data can exist almost anywhere.
+
+#### Local Front-Matter
+
+Use this if you just want a local variables. These are good for simple page properties. Add YAML front-matter to files to append properties to the file.
+
+*example.html*
+``` html
+---
+name: Linus Torvalds
+languages:
+- PHP
+- Perl
+- Python
+---
+<h1>Linus Torvalds</h1>
```

Comment:
I was a little confused to see that the variables are not used in the output. What's the point of the front matter then?